Aux module documentation (#12607)
rapid7/metasploit-framework: documentation/modules/auxiliary/scanner/afp/afp_login.md

## Vulnerable Application

> Apple Filing Protocol (AFP) is Apple's file sharing protocol similar to SMB, and NFS. This module attempts to bruteforce authentication credentials for AFP.

Reviewer: For consistency with `brute force` phrasing elsewhere.

```suggestion
Apple Filing Protocol (AFP) is Apple's file sharing protocol similar to SMB, and NFS. This module attempts to brute force authentication credentials for AFP.
```

References:

> [developer.apple.com](https://developer.apple.com/library/mac/documentation/Networking/Reference/AFP_Reference/Reference/reference.html)

Reviewer 2: I would rename this link to AFP_Reference

> [developer.apple/com](https://developer.apple.com/library/mac/documentation/networking/conceptual/afp/AFPSecurity/AFPSecurity.html)

Reviewer 2: I woudl rename this link to AFP_Security

> 1. `sudo apt-get install netatalk`

Reviewer 2: Can you add an L3 header to say what OS you did this on?
Exmaple:
```
### Kali 2019.3
```

2. edit `/etc/default/netatalk` and add the following lines:
  ```
  ATALKD_RUN=no
  PAPD_RUN=no
  CNID_METAD_RUN=yes
  AFPD_RUN=yes
  TIMELORD_RUN=no
  A2BOOT_RUN=no
```
3. Restart the service: `sudo /etc/init.d/netatalk restart`

## Verification Steps

  1. Start msfconsole
  2. Do: `use modules/auxiliary/scanner/afp/afp_login`

>   4. set RHOSTS [ip]

Reviewer: ```suggestion
  3. set RHOSTS [ip]
```

>   3. Do: `show options`

Reviewer 2: add a line for ```set rhosts [ip]``` instead of options

>   5. Do: `run`

Reviewer: ```suggestion
  4. Do: `run`
```

>       ...show and set options...

Reviewer 2: remove

  4. Do: `run`

## Scenarios

> 

Reviewer 2: Can you add an L3 header saying what os/version of the software this is.
Example:
```
### netatalk v2.1 on Kali 2019.3
```
